homeassistant/components/jvc_projector/entity.py

```diff
@@ -0,0 +1,38 @@
+"""Base Entity for the jvc_projector integration."""
+
+from __future__ import annotations
+
+import logging
+
+from jvcprojector import JvcProjector
+
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER, NAME
+from .coordinator import JvcProjectorDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class JvcProjectorEntity(CoordinatorEntity[JvcProjectorDataUpdateCoordinator]):
+    """Defines a base JVC Projector entity."""
```

Comment:
Required on new integrations.

```suggestion
    """Defines a base JVC Projector entity."""
    
    _attr_has_entity_name = True
```